### PR TITLE
Updated the picture

### PR DESCRIPTION
**Pull request by imfinedude**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by imfinedude** [comment]
updating the checklist

**Comment by imfinedude** [comment]
Thanks Shoaib

**Comment by Mohammed-Shoaib** [comment]
@imfinedude Your PR hasn't been merged yet but you closed it! :no_good: You will have to wait until a maintainer reviews it and merges it. So kindly reopen this PR :open_hands:

**Comment by imfinedude** [comment]
That was a mistake, reopened

File: _data/imfinedude/imfinedude.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Muhammad Q Shahzad # No longer than 18 characters
+institution: Syracuse University 🚩 # no longer than 58 characters
+profile_pic: imfinedude.png # Name and extension of your profile picture(ex. mona.png)
+quote: You will get to your destination as long as you keep going # no longer than 100 characters
+github_user: imfinedude
```
Comment (juanpflores) [inline _data/imfinedude/imfinedude.md:6]:
```suggestion
name: Muhammad Q Shahzad
institution: Syracuse University 🚩
profile_pic: imfinedude.png 
quote: You will get to your destination as long as you keep going
github_user: imfinedude
```

**Pull request merged**